src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -1307,13 +1307,12 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 		this._safeSetOption('macOptionClickForcesSelection', config.macOptionClickForcesSelection);
 		this._safeSetOption('rightClickSelectsWord', config.rightClickBehavior === 'selectWord');
 		this._safeSetOption('wordSeparator', config.wordSeparators);
-		if (config.rendererType === 'experimentalWebgl') {
+		if (config.rendererType === 'auto') {
```

Comment:
I think this is more correct?

```suggestion
		if (config.rendererType === 'auto' || config.rendererType === 'experimentalWebgl') {
```

Also do we have protections to avoid double enabling the webgl renderer?

Reply:
Oh I see there are protections with `if (!this._xterm || this._webglAddon) {`, all good

Reply:
Yep